src/pages/thanks-page/tests/thanks-page.test.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import { ThemeProvider } from '@material-ui/styles';
+
+import { BrowserRouter } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { theme } from '../../../components/app/app-theme/app.theme';
+import ThanksPage from '../thanks-page';
+import mockStore from './mockStore';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+const themeValue = theme('light');
+jest.mock('react-redux');
+
+const dispatch = jest.fn();
+const state = mockStore;
+
+useDispatch.mockImplementation(() => dispatch);
+useSelector.mockImplementation(() => state);
+
+let wrapper;
+
+describe('ThanksPage component tests', () => {
+  beforeEach(() => {
+    wrapper = mount(
```

Comment:
I think better to use shallow instead of mount, you can sync with your teammates they will explain you changes to be made.